site/sass/components/_header.phosphor.scss

```diff
@@ -0,0 +1,12 @@
+// Site Header
+
+.header {
+  border: solid var(--blue-850) 2px;
+  margin: 20px;
+  width: calc(100vw - 40px);
```

Comment:
Non-color items changing

Reply:
The nav specifically had inset/margin added in design for added impact (2/2)